content/applications/productivity/iot/devices/scale.rst

```diff
@@ -1,54 +1,56 @@
 ===============
-Connect a Scale
+Connect a scale
 ===============
 
-When using your **IoT Box** in Odoo, you could need to use a scale. Doing so is easy and convenient
-as it can be done in a few steps. Then, you can use it in your **Point of Sale app** to weigh your
-products, which is helpful if their price are based on it.
+A scale can be connected to the :abbr:`IoT (Internet of Things)` box in an Odoo database.
+Configuring one can be done in a few steps. After setup, the *Point of Sale* app can be used to
+weigh products, which is helpful if their prices are calculated based on weight.
 
 Connection
 ==========
 
-To link the scale to the **IoT Box**, connect them with a cable.
+To link the scale to the :abbr:`IoT (Internet of Things)` box, connect it with a :abbr:`USB
+(Universal Serial Bus)` cable.
 
 .. note::
-   In some cases, a serial to USB adapter may be needed.
+   In some cases, a serial port to :abbr:`USB (Universal Serial Bus)` adapter may be needed.
 
-If your scale is `compatible with Odoo IoT Box <https://www.odoo.com/page/iot-hardware>`_, there
-is no need to set up anything because it will be automatically detected as soon as it is connected.
+If the scale is `compatible with Odoo IoT Box <https://www.odoo.com/page/iot-hardware>`_, there is
+no need to set up anything because it will be automatically detected as soon as it is connected.
 
 .. image:: scale/iot-choice.png
    :align: center
    :alt: IOT box auto detection.
 
-You may need to restart the box and download your scales’ drivers from the box in some cases. To do
-so, go to the *IoT Box Home Page* and click on *drivers list*. Then, click on load drivers.
+The :abbr:`IoT (Internet of Things)` box may need to be restarted and the scale's drivers may need
+to be downloaded to the box in some cases. To do so, go to the :guilabel:`IoT Box Home Page` and
+click on :guilabel:`Drivers List`. Then, click on :guilabel:`Load Drivers`.
 
 .. image:: scale/driver-list.png
    :align: center
    :alt: View of the IoT box settings and driver list.
 
-Use a Scale in Point of Sale
-============================
+If loading the the drivers still doesn't allow for the scale to function it may be that the scale is
+not compatible with the Odoo :abbr:`IoT (Internet of Things)` box. In this case a different scale
+will need to be used.
```

Comment:
Deleting an extra word and adding commas.
```suggestion
If loading the drivers still doesn't allow for the scale to function, it may be that the scale is
not compatible with the Odoo :abbr:`IoT (Internet of Things)` box. In this case, a different scale
will need to be used.
```